public/docs/_examples/toh-6/dart/web/main.dart

```diff
@@ -0,0 +1,24 @@
+// #docplaster
+// #docregion final
+// #docregion v1
+import 'package:angular2/core.dart';
+import 'package:angular2/platform/browser.dart';
+import 'package:http/http.dart';
+import 'package:angular2_tour_of_heroes/app_component.dart';
+// #enddocregion v1
+import 'package:angular2_tour_of_heroes/in_memory_data_service.dart';
+
```

Comment:
I think the v1 imports should be:

```
import 'package:angular2/platform/browser.dart';
import 'package:angular2_tour_of_heroes/app_component.dart';
import 'package:http/browser_client.dart';
```

So no imports of `angular2/core.dart` or `http.dart`, (At least, that made WebStorm happy. The app still wouldn't run for me.)